admin/class-yoast-form.php

```diff
@@ -583,24 +590,27 @@ public function toggle_switch( $var, $values, $label ) {
 			$this->options[ $var ] = 'off';
 		}
 
+		$help_class = ! empty( $help ) ? ' switch-container__has-help' : '';
+
 		$var_esc = esc_attr( $var );
 
-		echo '<div class="switch-container">';
-		echo '<fieldset id="', $var_esc, '" class="fieldset-switch-toggle"><legend>', $label, '</legend>
-		<div class="switch-toggle switch-candy switch-yoast-seo">';
+		echo "<div class='switch-container$help_class'>";
```

Comment:
Please restore the double quotes for the HTML attribute.
This should be:
```
printf( '<div class="%s">', esc_attr( 'switch-container' . $help_class ) );
```

Reply:
OK, but I'm not sure it's really needed. The `$help_class` is hardcoded a few lines above and maybe doesn't need escaping?
`$help_class = ! empty( $help ) ? ' switch-container__has-help' : '';`